### ISSUE TITLE
Thumbnail preview is not clear when the thumbnail is big

### ISSUE THREAD

**Issue by kgfly** [nomacs/nomacs#844]
**Describe the bug**
Thumbnail preview is not clear when the thumbnail is big

**To Reproduce**
Open thumbnail preview.
Make the thumbnail bigger.
The image is not clear.

**Expected behavior**
When the thumbnail is big, the image should still be clear.

**Desktop (please complete the following information):**
 - OS: Windows
 - Version 3.16.1709

**Additional context**

**Comment by leucome** [comment]
Yeah it use the tiny emended thumbnails if they exist.   I tried to find a option to disable these and I could not find any.  So i guess it do not exist or maybe I missed it. 

SO I changed something in Dkthumbs.cpp  at line 102 to 113 to remove the ability to load MetaData thumbnails. 

	try {
		// [DIEM] READ  build crashed here 09.06.2016
		if (baZip && !baZip->isEmpty())	
			mImg = QImage();  //  metaData.readMetaData(filePath, baZip); 
		else if (!ba || ba->isEmpty())
			mImg = QImage();  //  metaData.readMetaData(filePath);
		else
			mImg = QImage();  //  metaData.readMetaData(filePath, ba);
		// read the full image if we want to create new thumbnails
		if (forceLoad != force_save_thumb)
			thumb = metaData.getThumbnail();
	}

Maybe  somebody who know this project better  could make a nice check box setting to do something like that.   Still until then it kinda works if somebody is interested to do the change and compile.
